### ISSUE TITLE
Missing suggestions for parent inherited variables inside closures

### ISSUE THREAD

**Issue by daltones** [Gert-dev/php-integrator-autocomplete-plus#51]
Hi! First of all, I want to thank you for all php-integrator-* packages. Brilliant work! :smiley:

Now the issue:

```php
$foo = new Foo();
// $foo->...   Suggestions here are okay.

$closure = function (Foo $bar) use ($foo) {
    // $bar->...   Suggestions here are okay, as $bar is a parameter of type Foo.

    // $foo->      No suggestions here! But for sure $foo is an instance of Foo.
};
```

**Comment by Gert-dev** [comment]
Thanks for the compliment! This was actually an issue in the base package. I've fixed it there and it should be fixed with 0.9.1. Thanks for taking the time to report this.

**Comment by daltones** [comment]
Wow, that was fast! Thanks a lot! :smiley: 
There’s only one more issue I found about closure use statement. I posted it there in the base package.
